### DISCUSSION TITLE
3D keypoints Unit ?

### DISCUSSION THREAD

**Discussion by HabibaAbderrahim** [open-mmlab/mmpose#2859] [Q&A]
For the coordinates provided for the 3D keypoints such as [-0.0, 0.0, 0.6314282417297363] , [0.10871679335832596, 0.07088106125593185, 0.6354875564575195] ect ... is it meters , cm  or any other unit ?

**Comment by bonor-ayambem** [comment]
Hi, I have the same question. Did you ever find an answer?
